src/web_analysis/data/prompt_templates.json

```diff
@@ -0,0 +1,44 @@
+{
+  "prompts": [
+    {
+      "id": "scraping-policy",
+      "content": "Question: Does the website restrict scraping? Carefully review the following Terms of Service (TOS) document and determine if the statement is True: {}. If the statement is True, provide the exact text from the TOS that answers the question as evidence. If the statement is False, set evidence as \"N/A\". Return only a dictionary with your verdict and the corresponding evidence. Format the response like this to ensure it can be loaded by JSON: {{\"verdict\": \"True\", \"evidence\": \"Exact text from TOS.\"}} or {{\"verdict\": \"False\", \"evidence\": \"N/A\"}}"
```

Comment:
Set evidence as "" (empty string) instead of N/A. 

Also, specify the evidence should be "one or more sentences"

Reply:
@shayne-longpre I initially had it returning an empty string and was getting a strange amount of JSON loading errors due to gpt returning incorrectly formatted responses. For some reason it's preforming better when I assign it a task (i.e. return "N/A") versus when I asked it to return nothing. In post-processing I convert the "N/A" to a None value